packages/notification-service/test/firebase.test.ts

```diff
@@ -47,7 +47,7 @@ describe('sendPaymentNotification', () => {
               "color": "#42D689",
               "icon": "ic_stat_rings",
             },
-            "priority": "high",
+            "priority": "normal",
```

Comment:
oh this means CI didn't catch this?
Looks like we might not be running other packages tests since we left the monorepo.

Reply:
Think so 😕

Reply:
I added a PR here to add those tests again: https://github.com/celo-org/wallet/pull/126